gnomad_qc/v4/create_release/validate_and_export_vcf.py

```diff
@@ -285,6 +286,14 @@ def select_type_from_joint_ht(ht: hl.Table, data_type: str) -> hl.Table:
     ht = ht.select(*row_fields)
     ht = ht.transmute_globals(**ht[f"{data_type}_globals"])
     ht = ht.transmute(**ht[data_type])
+    if "freq_index_dict" not in ht.globals.keys():
```

Comment:
This was a fix when the `freq_index_dict` wasn't present, it won't break, should we keet it?